code/AEPSDK.brs

```diff
@@ -46,11 +46,11 @@ end function
 function AdobeAEPSDKInit() as object
 
     if GetGlobalAA()._adb_public_api <> invalid then
-        _adb_logInfo("API: AdobeAEPSDKInit() - Please shut down the previous SDK instance before initializing a new one.")
+        _adb_logInfo("AdobeAEPSDKInit() - Please shut down the previous SDK instance before initializing a new one.")
```

Comment:
nit: "AdobeAEPSDKInit() - Unable to initialize SDK instance as there is an existing active instance. Call shutdown() API for existing instance before initializing a new one.